pynxtools/dataconverter/readers/ellips/mock.py

```diff
@@ -47,10 +47,10 @@ class MockEllips():
 
     def __init__(self, data_template) -> None:
         self.data = data_template["measured_data"]
-        self.wavelength = data_template["spectrometer/wavelength"]
+        self.wavelength = data_template["data_collection/NAME_spectrum[wavelength_spectrum]"]

```

Comment:
Is `data_collection` used in the appdef or it should rather be something like `DATA[data_collection]` here?

Reply:
`data_collection` is used in the app def. It's `data_collection(NXprocess)`.